source/_posts/2017-08-02-dht22.markdown

```diff
@@ -0,0 +1,214 @@
+---
+author: Michał Żygowski
+layout: post
+title: "1-Wire read data protocol on DHT22 sensor"
```

Comment:
@miczyg1 maybe something more human and SEO freindly: How to handle DHT22 sensor using ARM mbed OS ?

There may be something different but please consider what other people will write into search engine to find your post.